oap-ws-sso-api/src/main/java/oap/ws/sso/Authentication.java

```diff
@@ -27,22 +27,24 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import lombok.EqualsAndHashCode;
 import lombok.ToString;
+import oap.util.Pair;
 import org.joda.time.DateTime;
 
 import java.io.Serial;
 import java.io.Serializable;
+import java.util.Date;
 
 @ToString
 @EqualsAndHashCode
 public class Authentication implements Serializable {
     @Serial
     private static final long serialVersionUID = -2221117654361445000L;
-    public final String accessToken;
-    public final String refreshToken;
+    public final Pair<Date, String> accessToken;
```

Comment:
I would create a new class instead of pair. in order to easily change structure

Reply:
I would prefer to keep it like this. It's a standard approach for JWT.